style/buttons.css

```diff
@@ -13,4 +14,5 @@
     border: 1px solid var(--black);
     padding: 0.3em 0.6em;
     border-radius: 0 0.3rem 0.3rem 0;
+    cursor: pointer;
```

Comment:
Are you sure we need this as well?

Because it seems in every situation where you add  the `btn-navy` class you also have `btn` class